### PR TITLE
9477 QA user only reset first time

### PR DESCRIPTION
**Pull request by Sergio-Mira**
https://swrvedev.jira.com/browse/SWRVE-9477

### CODE REVIEW

**Comment by Sergio-Mira** [comment]
@dominicmarmionswrve @shanetmoore Could you have a look? Will write a test soon

File: SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java [hunk 1]
```diff
@@ -763,6 +763,8 @@ protected void loadCampaignsFromJSON(JSONObject json, JSONObject settings) {
 
             Map<Integer, String> campaignsDownloaded = null;
 
+            boolean wasQAUser = (qaUser != null);
```
Comment (shanetmoore) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java]: was?

**Comment by shanetmoore** [comment]
:+1: Syntactically looks fine. Just that question about the wasQAUser?

File: SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java (hunk 1 above)
Comment (Sergio-Mira) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java]: I can change it but wanted to reflect that we are checking it was a QA user already before we might make it one after this refresh (on that same method)

Comment (shanetmoore) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java]: Ah makes sense now. Could be worthy of a comment.

Comment (Sergio-Mira) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java]: I will change it to be "wasQAUserLastRefresh" maybe?

Comment (dominicmarmionswrve) [inline SwrveSDK/src/main/java/com/swrve/sdk/SwrveImp.java]: I prefer something like wasPreviouslyAQAUser.
Its not a nice name, so as Shane says, best to add a comment.

**Comment by dominicmarmionswrve** [comment]
Think it look ok. But needs tests.

**Pull request closed without merging**